src/InterlockingGenerator.cpp

```diff
@@ -71,10 +71,13 @@ std::pair<Polygons, Polygons> InterlockingGenerator::growBorderAreasPerpendicula
     Polygons from_border_a = a.difference(total_shrunk);
     Polygons from_border_b = b.difference(total_shrunk);
 
+    Polygons temp_a, temp_b;
     for (auto _ : ranges::view::iota(0, (detect / min_line) + 2))
```

Comment:
```suggestion
for (auto _ : ranges::views::iota(0, (detect / min_line) + 2))
```

`view` namespace is deprecated use `views` instead

Reply:
the includes do not change only the namespace
![image](https://user-images.githubusercontent.com/8535734/217506788-2264227d-db4b-4f7e-80fa-444fbe486e86.png)

Reply:
applied the review comment myself such that this can be merged

Reply:
thanks!